core/scripts/local/ExecuteAdmin.js

```diff
@@ -0,0 +1,35 @@
+// This executes an approved admin proposal. This assumes that the admin proposal is one to whitelist an identifier,
+// therefore it will check the identifier whitelist afterwards.
+
+const IdentifierWhitelist = artifacts.require("IdentifierWhitelist");
+const Governor = artifacts.require("Governor");
+const assert = require("assert");
+
+async function execute(callback) {
+  try {
+    const identifierWhitelist = await IdentifierWhitelist.deployed();
+    const governor = await Governor.deployed();
+
+    // Get the latest admin proposal
+    const proposalId = (await governor.numProposals()).subn(1).toString();
+    const proposal = await governor.getProposal(proposalId);
+
+    // for every transactions within the proposal
+    for (let i = 0; i < proposal.transactions.length; i++) {
+      console.log(`${i}: Submitting tx...`);
+      let tx = await governor.executeProposal(proposalId.toString(), i.toString());
```

Comment:
nit: `const`

Reply:
> Looks great! Thoughts on modifying the votes in the resolved requests table so they say YES for nonzero values and NO for zero values?

Ah yes, will do!

Reply:
> Looks great! Thoughts on modifying the votes in the resolved requests table so they say YES for nonzero values and NO for zero values?

Done here. Also in the process of testing this I was able to test out the new feature where you can't retrieve rewards if you voted incorrectly!
![Screen Shot 2020-07-01 at 15 46 06](https://user-images.githubusercontent.com/9457025/86285183-26ef2100-bbb2-11ea-8c88-bc3f2bee54ef.png)

Reply:
very nice :)